docs/tutorial.rst

```diff
@@ -709,13 +709,17 @@ non-default limit. This reaction is an ATP maintenance reaction and the
 modelers chose to force a certain level of flux through it to simulate the
 general energy cost of cellular maintenance.
 
-Medium
-~~~~~~
+Exchange compounds
+~~~~~~~~~~~~~~~~~~
 
-The medium file is where you can designate the boundary conditions for the
-model. The compartment of the medium compounds can be designated using the
+The exchange file is where you can designate the boundary conditions for the
+model. This includes both compounds that are allowed to enter the model system
+(*the medium*) as well as compounds that are allowed to exit the model.
+Settings a negative lower limit on an exchange compounds allows an uptake of
+that compound thereby making the compound part of the medium. The
```

Comment:
I would like to explain that negative values correspond to uptake since I don't think we explain this anywhere else. There is also a sentence later in this document: "For example if glucose was added into the medium along with mannitol then the results might appear as follows" where "added into the medium" specifically means that the lower bound was set to a negative value. For that to make sense the medium has to be the set of exchange compounds with negative lower bound. Maybe we can change that sentence to spell out that the lower bound is set negative in the exchange file instead of saying that it was "added into the medium"? What do you think?